src/layout/section/heroSection/story.tsx

```diff
@@ -1,76 +1,93 @@
 import React from 'react'
-import { text, withKnobs } from '@storybook/addon-knobs'
+import { select, text, withKnobs } from '@storybook/addon-knobs'
```

Comment:
Since you are updating this component would be great to migrate its story 👼 
Especially removing "withKnobs" from this declaration since we don't need it anymore